src/documentation/io/authoring-overview.md

```diff
@@ -10,35 +10,76 @@ permalink: /documentation/io/authoring-overview/
 
 _A guide for users who need to connect to a data store that isn't supported by the [Built-in I/O Transforms]({{site.baseurl }}/documentation/io/built-in/)_
 
-> Note: This guide is still in progress. There is an open issue to finish the guide: [BEAM-1025](https://issues.apache.org/jira/browse/BEAM-1025).
 
 * TOC
 {:toc}
 
 ## Introduction
-TODO
+This is a guide to implementing read and write transforms in the Beam model. These transforms are the way that Beam pipelines import data so it can be processed and then write data to a store.
+
+Reading/writing data in Beam is just a parallel task like everything else. Most of the time you can assemble it from ParDo’s, GBK’s etc.; rarely you will need the more specialized Source and Sink classes for specific features. There are changes coming soon (SplittableDoFn, BEAM-65) that will make Source and Sink unnecessary.
```

Comment:
is there significance in using the word "assemble"? did some compaction of first part:
Reading and writing data in Beam is a parallel task, and using `ParDo`s, `GroupByKey`s, etc. is usually sufficient. Rarely, you will need ...